### ISSUE TITLE
Unable to setup. 

### ISSUE THREAD

**Issue by boranhan** [ServiceNow/context-is-key-forecasting#2]
Dear authors, 

I run into an error creating the environment. These are my command lines:

```
python -m venv venv/LLMforecasting
source venv/LLMforecasting/bin/activate
git clone https://github.com/ServiceNow/context-is-key-forecasting.git
cd context-is-key-forecasting/
pip install -e .
```

The error message is shown as below: 

```
Obtaining file:///home/ubuntu/fsx/context-is-key-forecasting                                                                                                                  
  Installing build dependencies ... done                                                                                                                                      
  Checking if build backend supports build_editable ... done                                                                                                                  
  Getting requirements to build editable ... error                                                                                                                            
  error: subprocess-exited-with-error                                                                                                                                         
                                                                                                                                                                              
  × Getting requirements to build editable did not run successfully.                                                                                                          
  │ exit code: 1                                                                                                                                                              
  ╰─> [30 lines of output]                                                                                                                                                    
      Traceback (most recent call last):                                                                                                                                      
        File "/home/ubuntu/fsx/venv/LLMforecasting/lib/python3.10/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 353, in <module>                
          main()                                                                                                                                                              
        File "/home/ubuntu/fsx/venv/LLMforecasting/lib/python3.10/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 335, in main                    
          json_out['return_val'] = hook(**hook_input['kwargs'])                                                                                                               
        File "/home/ubuntu/fsx/venv/LLMforecasting/lib/python3.10/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 132, in get_requires_for_build_e
ditable                                                                                                                                                                       
          return hook(config_settings)                                                                                                                                        
        File "/tmp/pip-build-env-gdkuzo4o/overlay/lib/python3.10/site-packages/setuptools/build_meta.py", line 482, in get_requires_for_build_editable                        
          return self.get_requires_for_build_wheel(config_settings)                                                                                                           
        File "/tmp/pip-build-env-gdkuzo4o/overlay/lib/python3.10/site-packages/setuptools/build_meta.py", line 333, in get_requires_for_build_wheel                           
          return self._get_build_requires(config_settings, requirements=[])                                                                                                   
        File "/tmp/pip-build-env-gdkuzo4o/overlay/lib/python3.10/site-packages/setuptools/build_meta.py", line 303, in _get_build_requires                                    
          self.run_setup()                                                                                                                                                    
        File "/tmp/pip-build-env-gdkuzo4o/overlay/lib/python3.10/site-packages/setuptools/build_meta.py", line 319, in run_setup                                              
          exec(code, locals())                                                                                                                                                
        File "<string>", line 1, in <module>                                                                                                                                  
        File "/tmp/pip-build-env-gdkuzo4o/overlay/lib/python3.10/site-packages/setuptools/__init__.py", line 117, in setup                                                    
          return distutils.core.setup(**attrs)                                                                                                                                
        File "/tmp/pip-build-env-gdkuzo4o/overlay/lib/python3.10/site-packages/setuptools/_distutils/core.py", line 157, in setup                                             
          dist.parse_config_files()                                                                                                                                           
        File "/tmp/pip-build-env-gdkuzo4o/overlay/lib/python3.10/site-packages/setuptools/dist.py", line 655, in parse_config_files                                           
          pyprojecttoml.apply_configuration(self, filename, ignore_option_errors)                                                                                             
        File "/tmp/pip-build-env-gdkuzo4o/overlay/lib/python3.10/site-packages/setuptools/config/pyprojecttoml.py", line 71, in apply_configuration                           
          config = read_configuration(filepath, True, ignore_option_errors, dist)                                                                                             
        File "/tmp/pip-build-env-gdkuzo4o/overlay/lib/python3.10/site-packages/setuptools/config/pyprojecttoml.py", line 139, in read_configuration                           
          validate(subset, filepath)                                                                                                                                          
        File "/tmp/pip-build-env-gdkuzo4o/overlay/lib/python3.10/site-packages/setuptools/config/pyprojecttoml.py", line 60, in validate                                      
          raise ValueError(f"{error}\n{summary}") from None                                                                                                                   
      ValueError: invalid pyproject.toml config: `project`.                                                                                                                   
      configuration error: `project` must not contain {'long_description_content_type'} properties                                                                            
      [end of output]                                                                                                                                                         
   
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error
```

I'm using `Python 3.10.9`. I'm wondering if you help me resolve it? Thanks

**Comment by boranhan** [comment]
I also tried `pip install -r requirements.txt` and run `python run_baselines.py --exp-spec myfile.json`, However, it said `ModuleNotFoundError: No module named 'benchmark'`.

**Comment by aldro61** [comment]
Hi @boranhan! Thanks for flagging this and being our first user :) This should be solved in #3 and #4. Let us know if not.
